2019-02-08 12:21:47,465 ERROR: Exception on /index [GET] [in /home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/kiborgok/parrotSc/mmu/venv/src/flask-login/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/kiborgok/parrotSc/mmu/app/routes.py", line 27, in index
    return render_template('index.html', title='Home', form=form, posts=posts.items, next_url=next_url, prev_url=prev_url)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/templating.py", line 135, in render_template
    context, ctx.app)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/templating.py", line 117, in _render
    rv = template.render(context)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/kiborgok/parrotSc/mmu/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/kiborgok/parrotSc/mmu/app/templates/base.html", line 36, in top-level template code
    {{ super() }}
  File "/home/kiborgok/parrotSc/mmu/app/templates/base.html", line 37, in block "scripts"
    {{ moment.include_moment() }}
jinja2.exceptions.UndefinedError: there is no parent block called 'scripts'.
2019-02-08 12:22:05,374 ERROR: Exception on /index [GET] [in /home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/kiborgok/parrotSc/mmu/venv/src/flask-login/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/kiborgok/parrotSc/mmu/app/routes.py", line 27, in index
    return render_template('index.html', title='Home', form=form, posts=posts.items, next_url=next_url, prev_url=prev_url)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/templating.py", line 135, in render_template
    context, ctx.app)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/templating.py", line 117, in _render
    rv = template.render(context)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/kiborgok/parrotSc/mmu/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/kiborgok/parrotSc/mmu/app/templates/base.html", line 36, in top-level template code
    {{ moment.include_moment() }}
  File "/home/kiborgok/parrotSc/mmu/app/templates/base.html", line 37, in block "scripts"
    {% endblock %}
jinja2.exceptions.UndefinedError: there is no parent block called 'scripts'.
2019-02-08 12:22:23,055 ERROR: Exception on /explore [GET] [in /home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/kiborgok/parrotSc/mmu/venv/src/flask-login/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/kiborgok/parrotSc/mmu/app/routes.py", line 38, in explore
    return render_template("index.html", title='Explore', posts=posts.items, next_url=next_url, prev_url=prev_url)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/templating.py", line 135, in render_template
    context, ctx.app)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/flask/templating.py", line 117, in _render
    rv = template.render(context)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/kiborgok/parrotSc/mmu/venv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/kiborgok/parrotSc/mmu/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/kiborgok/parrotSc/mmu/app/templates/base.html", line 36, in top-level template code
    {% endblock %}
  File "/home/kiborgok/parrotSc/mmu/app/templates/base.html", line 37, in block "scripts"
    </body>
jinja2.exceptions.UndefinedError: there is no parent block called 'scripts'.
2019-02-08 12:23:22,926 INFO: MMUblog startup [in /home/kiborgok/parrotSc/mmu/app/__init__.py:49]
2019-02-08 12:35:10,502 INFO: MMUblog startup [in /home/kiborgok/parrotSc/mmu/app/__init__.py:50]
2019-02-08 12:52:17,116 INFO: MMUblog startup [in /home/kiborgok/parrotSc/mmu/app/__init__.py:50]
2019-02-08 12:54:15,680 INFO: MMUblog startup [in /home/kiborgok/parrotSc/mmu/app/__init__.py:50]
